RIOT-OS/RIOT
# tests/pkg/lwip: port for IPv4 testing

### Contribution description
This adapts the test script to also test IPv4 support. It checks the same environment variables used to compile the different IP versions.



### Testing procedure
```bash
for ipv6 in 0 1; do
    for ipv4 in 0 1; do
        if [ ${ipv6} == 0 ] && [ ${ipv4} == 0 ]; then
            continue
        fi
        LWIP_IPV4=${ipv4} LWIP_IPV6=${ipv6} QUIET=1 QUIETER=1 make -C tests/pkg/lwip flash test \
          || echo "LWIP_IPV4=${ipv4} LWIP_IPV6=${ipv6} make -C tests/pkg/lwip flash test errored!" >&2
    done
done
```

should not show any error message.



### Issues/PRs references
None
